SurfaceTopography/Uniform/common.py

```diff
@@ -150,10 +149,15 @@ def derivative(topography, n, scale_factor=1, operator=None, periodic=None,
         Surface topography object containing height information.
     n : int
         Order of the derivative.
-    scale_factor : int or list of ints, optional
+    scale_factor : int or list of ints or list of tuples of ints, optional
```

Comment:
Is it sufficiently clear, what giving a "list of tuples of ints"  means here in terms of a scale factor? 
It's a rather complex argument value.

Reply:
I extended the docstring to explain what the tuples do